src/cmdlib.sh

```diff
@@ -484,7 +484,8 @@ EOF
 
     cachedisk=()
     if [ -f "${workdir}/cache/cache.qcow2" ]; then
-        cachedisk=("-hdb" "${workdir}/cache/cache.qcow2")
+        cachedisk=("-drive" "if=none,id=drive-scsi0-0-0-1,discard=unmap,file=${workdir}/cache/cache.qcow2" \
```

Comment:
Random thought...do we still need the cache disk versus relying on a 9p mount?  I may investigate this.
Would simplify things a lot if we had just one disk attached to the VM versus two.

Reply:
That was the original design, but we moved away from that because composing directly into 9p caused issues. And composing on the side and moving cache/data back and forth was a lot of overhead.

Reply:
Check out https://github.com/coreos/coreos-assembler/pull/124#issuecomment-434040407 and following.